package.json

```diff
@@ -115,58 +115,37 @@
     "packages/*/index.js",
     "packages/*/lib/**/*",
     "packages/*/src/**/*",
+    "packages/node_modules/**/*.js",
+    "packages/node_modules/**/*.d.ts",
+    "packages/node_modules/**/*.wasm",
+    "packages/node_modules/**/LICENSE",
```

Comment:
We should make sure that LICENSE files that has an extension (e.g. `LICENSE.md` are also included)